web/src/components/vis/ScreePlot.vue

```diff
@@ -1,20 +1,40 @@
-<template lang="pug">
-.main(v-resize:throttle="onResize")
-  svg(ref="svg", :width="width", :height="height", xmlns="http://www.w3.org/2000/svg")
-    g.master
-      g.axes
-      g.label.x
-        text {{ xlabel }}
-      g.label.y
-        text {{ ylabel }}
-      g.plot
-        path.line
-        g.points
-        g.cutoffs
-          line.cutoff.cutoff50
-          line.cutoff.cutoff80
-          line.cutoff.cutoff90
-  .tooltip(ref="tooltip")
+<template>
+  <div
+    ref="mainRef"
+    class="main"
+  >
+    <!-- v-resize:throttle="onResize"-->
```

Comment:
None of the resize controls work anymore.    I feel like we should just get rid of all those zoom controls and the needlessly complex layout styles that drive them.  flex boxes and no resizing would probably be fine.

![Screenshot from 2020-07-20 08-59-46](https://user-images.githubusercontent.com/4214172/87940409-892b9b00-ca67-11ea-8c35-a47829c906a0.png)

Reply:
Agreed.